apps/wallet-mobile/src/features/RampOnOff/useCases/CreateExchange/CreateExchange.tsx

```diff
@@ -61,10 +62,10 @@ export const CreateExchange = () => {
       blockchain: 'ADA',
       walletAddress,
       returnUrl,
-    } as Banxa.ReferralUrlQueryStringParams
+    } as Exchange.ReferralUrlQueryStringParams
 
-    const banxa = banxaModuleMaker(moduleOptions)
-    const url = banxa.createReferralUrl(urlOptions)
+    const banxa = exchangeModuleMaker(moduleOptions)
+    const url = banxa.createReferralUrl('banxa', urlOptions)
```

Comment:
@stackchain  Finally i have opted for this solution instead of adding to the params.

let me know if you think otherwise